DependencyInjection/DoctrineExtension.php

```diff
@@ -761,6 +763,21 @@ private function loadPropertyInfoExtractor($entityManagerName, ContainerBuilder
         $propertyExtractorDefinition->addTag('property_info.type_extractor', ['priority' => -999]);
     }
 
+    /**
+     * Loads a validator loader for each defined entity manager.
+     */
+    private function loadValidatorLoader(string $entityManagerName, ContainerBuilder $container) : void
+    {
+        if (!class_exists(DoctrineLoader::class)) {
```

Comment:
This should fix the build:
```suggestion
        if (! class_exists(DoctrineLoader::class)) {
```